sbi/analysis/tensorboard_output.py

```diff
@@ -175,9 +175,11 @@ def _get_event_data_from_log_dir(
                 # we inspect their argument signature. These events are named tuples
                 # that can be found here:
                 # https://github.com/tensorflow/tensorboard/blob/b84f3738032277894c6f3fd3e011f032a89d002c/tensorboard/backend/event_processing/event_accumulator.py#L37
+                # When looping over `inspect.getfullargspec()` there is also a `self`
+                # attribute.
                 _type = type(data[0])
                 for attribute in inspect.getfullargspec(_type).args:
-                    if not attribute.startswith("_"):
+                    if not attribute.startswith("_") and not attribute == "self":
```

Comment:
Not sure when this happened but probably when tensorboard changed their event accumulator api from namedtuples to dataclass (which then causes the additional "self" attribute in the argspec). So this fix would do it! 👍 
For reference, old event accumulator api https://github.com/tensorflow/tensorboard/blob/b84f3738032277894c6f3fd3e011f032a89d002c/tensorboard/backend/event_processing/event_accumulator.py#L37 
and new api
https://github.com/tensorflow/tensorboard/blob/517a57e1c188f66006d201427e599b908465f5ba/tensorboard/backend/event_processing/event_accumulator.py#L43

Reply:
Ah, nice, thanks for the references!